docs/gossip.md

```diff
@@ -23,4 +24,82 @@ If you are using [bastion hosts](bastion.md), it is a bit tricky to find the bas
 
 ```
 kops toolbox dump -ojson | grep 'bastion.*elb.amazonaws.com'
-```
\ No newline at end of file
+```
+
+## Switching to memberlist from weaveworks mesh
+
+Weaveworks mesh is known to spike exponentially with CPU, memory and network bandwidth usage on cluster scale up. [issue 7429](https://github.com/kubernetes/kops/issues/7427), [issue 7436](https://github.com/kubernetes/kops/issues/7436), [issue 13974](https://github.com/kubernetes/kops/issues/13974)
+
+### Turn on memberlist as secondary protocol
+
+Add `gossipConfig` to your cluster configuration:
+
+```bash
+kops edit cluster
+```
+
+```yaml
+  gossipConfig:
+    listen: 0.0.0.0:3999
+    protocol: mesh
+    secondary:
+      listen: 0.0.0.0:4000
+      protocol: memberlist
+
+  dnsControllerGossipConfig:
+    protocol: mesh
+    listen: 0.0.0.0:3998
+    seed: 127.0.0.1:3999
+    secondary:
+      protocol: memberlist
+      listen: 0.0.0.0:3993
```

Comment:
Could these two be put in consistent order?

Reply:
Of course